donkeycar/utilities/TrackSpeedPlanner/uninstall-pi.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
I think this file should be deleted as there is nothing to uninstall.

Reply:
deleted